### PR TITLE
additional-tests-for-adapter

### PR DESCRIPTION
**Pull request by michmuel**
Closes https://github.com/openoereb/pyramid_oereb/issues/1880

### CODE REVIEW

**Comment by svamaa** [comment]
> Options:
> 
> * change the scope of the fixture, and explicitly include the function in 20-30 tests that require this patch
> * move the adapter tests in a folder on a higher directory level. Structure of code and tests would differ in this case.
> 
> What shall we do?

How would you estimate the payload for the first option? It seems to me to be the cleaner option but I'm not sure if it exceeds the goal of this ticket?

**Comment by michmuel** [comment]
I guess the first option is doable. Seems to be almost only copy and paste.

**Comment by michmuel** [comment]
I had again a look at 
https://github.com/openoereb/pyramid_oereb/blob/c8201eec7848cf6f655aa9cb63f4dffd7e3586ba/tests/conftest.py#L228
Balancing effort and yield I moved the adapter tests into a separate directory.

File: tests/core_adapter/test_adapter_database.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    adapter = DatabaseAdapter()
+    adapter.add_connection(db_url)
+    assert 'Connection already exists: {0}'.format(db_url) not in caplog.text
+    adapter.add_connection(db_url)
+    assert 'Connection already exists: {0}'.format(db_url) in caplog.text
+
+
+def test_get_connections():
+    db_url = 'postgresql://adapter_user:adapter_pswd@adapter-tests-02:5432/adapter-db'
+    test_connection = {db_url: {"engine": "test_engne", "session": "test_session"}}
```
Comment (voisardf) [inline tests/core_adapter/test_adapter_database.py:36]: small typo: test_engne instead of test_engine - not an issue for the test

**Review by voisardf — COMMENTED** [review]
Looks good to me and reorganised tests pass in my local instance

File: tests/core_adapter/test_adapter_database.py (hunk 1 above)
Comment (michmuel) [reply tests/core_adapter/test_adapter_database.py]: Corrected.

**Review by svamaa — APPROVED** [review]
Looks good to me! 
Thanks for the work!

**Review by voisardf — APPROVED** [review]
sorry I didn't answer earlier. Still ok for me, thanks 👍

**Pull request merged**